pagopa/io-app
# chore: [IA-916] Fix build errors on Apple silicon

## Short description
This PR makes the app build correctly on Mac with Apple silicon. Especially on iOS, the app for the simulator needs to be built using `x86_64` architecture.

## List of changes proposed in this pull request
- Update Flipper to 0.154 to fix a build error
- Remove `arm64` from supported architectures used for building the app for the simulator (this is needed because `vision-camera` doesn't support Apple silicon)

## How to test
Make sure you have Rosetta installed, then build the app on both iOS and Android. This shouldn't change anything on Macs with Intel.
